Task3/js/components/Zombie.js

```diff
@@ -0,0 +1,59 @@
+function Zombie(zombie) {
+    this.zombie = zombie || null;
```

Comment:
1) zombie is just a model
So, it shouldn't has any methods or additional properties.
BTW, what's the point of "|| null"? Will this help in any case?

2) what's the point of "this.zombie"?
You already have "zombie" model to get data and it's available in any function below
You have "this" to define methods or properties